common/src/ui/MapDocument.cpp

```diff
@@ -4387,6 +4387,19 @@ void MapDocument::clearWorld()
   m_currentLayer = nullptr;
 }
 
+void MapDocument::setWorldDefaultProps()
```

Comment:
I think this should be a free function in an anonymous namespace. There already are too many functions in the `MapDocument` interface.

Reply:
Sure, I'll look into this.